packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbarToggleGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import styles from '@patternfly/react-styles/css/components/DataToolbar/data-toolbar';
+import { css, getModifier } from '@patternfly/react-styles';
+import { DataToolbarGroupProps } from './DataToolbarGroup';
+import { DataToolbarContext } from './DataToolbarUtils';
+import { Button } from '../../../components/Button';
+
+import {
+  DataToolbarBreakpointMod,
+  DataToolbarSpacer,
+  formatBreakpointMods,
+  formatSpacers
+} from './DataToolbarUtils';
+
+export interface DataToolbarToggleGroupProps extends DataToolbarGroupProps {
+  /** An Icon to be rendered when the toggle group has collapsed down */
+  toggleIcon: React.ReactNode;
+  /** The breakpoint at which the toggle group is collapsed down */
+  breakpoint: 'md' | 'lg' | 'xl' | '2xl';
+}
+
+export class DataToolbarToggleGroup extends React.Component<DataToolbarToggleGroupProps> {
+
+  static defaultProps = {
+    breakpointMods: [] as DataToolbarBreakpointMod[],
+    spacers: [] as DataToolbarSpacer[],
+  };
+
+  render() {
+    const { toggleIcon, breakpoint, variant, breakpointMods, spacers, className, children, ...props } = this.props;
+
+    return (
+      <DataToolbarContext.Consumer>
```

Comment:
I forgot to mention this before, and you can take this or leave it, but there is a less verbose way to consume Context if you only need type of Context in your component.

Instead of rendering `<DataToolbarContext.Consumer>` with a render prop child, you could change this to a regular render method that pulls these values out from `this.context` like `const { isExpanded, toggleIsExpanded, expandableContentRef, expandableContentId } = this.context`, as long as you add `static contextType = DataToolbarContext;` up there next to your defaultProps.